frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -127,11 +127,11 @@ const PVCTableRow = ({ obj, index, key, style }) => {
         {obj?.spec?.storageClassName ? (
           <ResourceLink
             kind="StorageClass"
-            name={obj?.spec?.storageClassName}
-            title={obj?.spec?.storageClassName}
+            name={obj.spec.storageClassName}
+            title={obj.spec.storageClassName}
           />
         ) : (
-          <div className="text-muted">-</div>
+          <div>-</div>
```

Comment:
This div should be unnecessary.

```suggestion
          <>-</>
```

Reply:
> This div should be unnecessary.

Got it, thanks